PIL/Image.py

```diff
@@ -1092,12 +1154,14 @@ def draft(self, mode, size):
         pass
```

Comment:
The signature could probably be `(Text, Tuple[int, ...]) -> Something`...but why does the docstring say it's implemented for JPEG && PCD images when it's just a `pass`?